docs/Reference/GLOSSARY.md

```diff
@@ -0,0 +1,41 @@
+# GLOSSARY
+
```

Comment:
would love to help develop this glossary! 
For style only (not substantive use or description) we are using this organic [Word List Cosmo](https://docs.google.com/spreadsheets/d/15raX8lfzzeeifGrDiXSFTHaEDbsMVlFJzKgnNelihjs/edit#gid=0) in partnership with B9lab (the consulting firm doing a learning portal for Cosmos SDK). Part of https://github.com/cosmos/advocates/issues/23 in the Cosmos SDK advocates repo